apps/estuary/lib/estuary/data_writer.ex

```diff
@@ -0,0 +1,51 @@
+defmodule Estuary.DataWriter do
+  @moduledoc """
+  Implementation of `Pipeline.Writer` for Estuary's edges.
+  """
+
+  alias Estuary.Datasets.DatasetSchema
+
+  @behaviour Pipeline.Writer
+
+  @table_writer Application.get_env(:estuary, :table_writer)
+
+  alias Estuary.Datasets.DatasetSchema
+
+  @impl Pipeline.Writer
+  @doc """
+  Ensures a table exists using `:table_writer` from
+  Estuary's application environment.
+  """
+
```

Comment:
I don't think there should be a newline between the @impl and docs and the method they're annotating